src/tasks/helpers/base.ts

```diff
@@ -26,6 +26,15 @@ function isAppBuild<T extends PackageTaskTitaniumBuildBase>(definition: PackageT
 	return false;
 }
 
+function shouldEnableLiveview (definition: AppBuildTaskTitaniumBuildBase, context: TaskExecutionContext): boolean {
```

Comment:
<!--
  0 failure: 
  1 warning:  src/tasks/helpers...
  
  
  DangerID: danger-id-Danger;
  File: src/tasks/helpers/base.ts;
  Line: 29;
-->

- :warning: src/tasks/helpers/base.ts line 29 – 'context' is defined but never used. (@typescript-eslint/no-unused-vars)